src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets

```diff
@@ -29,37 +29,67 @@ Copyright (c) .NET Foundation. All rights reserved.
     automatically selects the version of .NET Core to target.
 
     The framework version that is written to the runtimeconfig.json file is based on the actual resolved package version
-    of Microsoft.AspNetCore.App or Microsoft.AspNetCore.All. This is to allow floating the verion number (i.e. the user
+    of Microsoft.AspNetCore.App or Microsoft.AspNetCore.All. This is to allow floating the version number (i.e. the user
     could set the version to "2.0-*".
 
   -->
   <Choose>
     <When Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'" >
 
+      <!-- 
+        These properties will allow the default and latest patch versions to be set in the CLI (via BundledVersions.props) 
+        or overridden by tests. The latest patch version for 2.1 uses a while card so self-contained deployment gets the 
+        latest stable metapackage.
+      -->
+      <PropertyGroup>
+        <DefaultPatchVersionForAspNetCoreAll2_1 Condition="'$(DefaultPatchVersionForAspNetCoreAll2_1)' == ''">2.1.1</DefaultPatchVersionForAspNetCoreAll2_1>
+        <DefaultPatchVersionForAspNetCoreApp2_1 Condition="'$(DefaultPatchVersionForAspNetCoreApp2_1)' == ''">2.1.1</DefaultPatchVersionForAspNetCoreApp2_1>
+        
+        <LatestPatchVersionForAspNetCoreAll2_1 Condition="'$(LatestPatchVersionForAspNetCoreAll2_1)' == ''">2.1.*</LatestPatchVersionForAspNetCoreAll2_1>
+        <LatestPatchVersionForAspNetCoreApp2_1 Condition="'$(LatestPatchVersionForAspNetCoreApp2_1)' == ''">2.1.*</LatestPatchVersionForAspNetCoreApp2_1>
+      </PropertyGroup>
+
+      <PropertyGroup Condition="'$(DefaultPatchVersionForAspNetCoreAll2_2)' == ''">
+        <DefaultPatchVersionForAspNetCoreAll2_2>2.2.0</DefaultPatchVersionForAspNetCoreAll2_2>
+        <!-- Check the bundled version. It will start with 2.2.0 until create the final version of the 2.2.0 runtimes.-->
+        <DefaultPatchVersionForAspNetCoreAll2_2 Condition="$(BundledAspNetCoreAllPackageVersion.StartsWith('2.2.0'))">$(BundledAspNetCoreAllPackageVersion)</DefaultPatchVersionForAspNetCoreAll2_2>
```

Comment:
I'd suggest using a dash here (and in the below condition for AspNetCoreApp), as I did for https://github.com/dotnet/cli/pull/9374.  It's not needed here for correctness since we won't release an "01" patch version, but ensures that if we update the number or copy-paste it somewhere else it's more likely to be correct.